index.js

```diff
@@ -2,35 +2,38 @@
 // file uses "T" to mean "Type byte", "TF" to mean "Type byte and Format byte"
 // and "D" to mean "Data bytes".
 
-const STRING_TF = Buffer.from([6, 0])
-const BOOL_TF = Buffer.from([6, 1])
-const BOOL_TRUE = Buffer.from([1])
-const BOOL_FALSE = Buffer.from([0])
-const NIL_TF = Buffer.from([6, 2])
-const NIL_TFD = NIL_TF
-
 const FEED_T = Buffer.from([0])
 const CLASSIC_FEED_TF = Buffer.from([0, 0])
-const GG_FEED_TF = Buffer.from([0, 1])
-const BB_FEED_TF = Buffer.from([0, 3])
+const GABBYGR_FEED_TF = Buffer.from([0, 1])
+const BENDYBT_FEED_TF = Buffer.from([0, 3])
 
 const MSG_T = Buffer.from([1])
 const CLASSIC_MSG_TF = Buffer.from([1, 0])
-const GG_MSG_TF = Buffer.from([1, 1])
-const BB_MSG_TF = Buffer.from([1, 4])
+const GABBYGR_MSG_TF = Buffer.from([1, 1])
+const BENDYBT_MSG_TF = Buffer.from([1, 4])
+
+const BLOB_T = Buffer.from([2])
+const CLASSIC_BLOB_TF = Buffer.from([2, 0])
 
 const SIGNATURE_TF = Buffer.from([4, 0])
 
 const BOX_T = Buffer.from([5])
 const BOX1_TF = Buffer.from([5, 0])
 const BOX2_TF = Buffer.from([5, 1])
 
+const STRING_TF = Buffer.from([6, 0])
+const BOOL_TF = Buffer.from([6, 1])
+const BOOL_TRUE = Buffer.from([1])
+const BOOL_FALSE = Buffer.from([0])
+const NIL_TF = Buffer.from([6, 2])
+const NIL_TFD = NIL_TF
+
 const encoder = {
   feed(feedId) {
     let tf
     if (feedId.endsWith('.ed25519')) tf = CLASSIC_FEED_TF
-    else if (feedId.endsWith('.bbfeed-v1')) tf = BB_FEED_TF
-    else if (feedId.endsWith('.ggfeed-v1')) tf = GG_FEED_TF
+    else if (feedId.endsWith('.ggfeed-v1')) tf = GABBYGR_FEED_TF
+    else if (feedId.endsWith('.bbfeed-v1')) tf = BENDYBT_FEED_TF
```

Comment:
Reordered just to match the same order as in `F`.

Reply:
Nice. Very satisfying to see all three with the same length (`CLASSIC`, `GABBYGR` and `BENDYBT`). I am learning how your mind works :)